pkg/util/limiter/utilization.go

```diff
@@ -144,6 +144,8 @@ func (l *UtilizationBasedLimiter) compute(nowFn func() time.Time) (currCPUUtil f
 	// Add the instant CPU utilization to the moving average. The instant CPU
 	// utilization can only be computed starting from the 2nd tick.
 	if prevUpdate, prevCPUTime := l.lastUpdate, l.lastCPUTime; !prevUpdate.IsZero() {
+		// Note that `time.Sub` is robust against wall clock changes, since it's based on the monotonic clock:
+		// https://pkg.go.dev/time#hdr-Monotonic_Clocks
```

Comment:
That's only true if `nowFn()` returns time with monotonic clock reading.

Reply:
Good point, Peter. I will revise accordingly.

Reply:
Done.